### ISSUE TITLE
Campaigns Broken.

### ISSUE THREAD

**Issue by iwantaspeaker** [mautic/mautic#6612]
**Please DO NOT report security vulnerabilities here. Send them to security@mautic.com instead.**

What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Yes
| Feature request? | 
| Enhancement? | 

## Description:
I am having problems with campaigns progressing.
The campaign was due to send 4 emails, 7 days apart to 4826 members. First emailing sent to 83 (out of 4826), 2nd email sent to 619 and the 3rd email has gone to 590.
IMAGE: https://www.dropbox.com/s/04a3bakbhcnwbn8/Screenshot%20%2856%29.png?dl=0

I'm seeing the same problem on another campaign aswell.
The campaign simply checks a segment of bounced emails, to make sure the email is valid. Then changes the Tag depending whether it is correct or not.

I am getting the following in the Cron logs:
`Triggering events for campaign 14
Triggering events for newly added contacts
144 total events(s) to be processed in batches of 100 contacts
   0/144 [>---------------------------]   0%
 144/144 [============================] 100%

0 total events were executed
0 total events were scheduled

Triggering scheduled events
0 total events(s) to be processed in batches of 100 contacts
0 total events were executed
0 total events were scheduled

Triggering events for inactive contacts

0 total events were executed
0 total events were scheduled`

and the Campaign "Actions" Tab shows as:
IMAGE: https://www.dropbox.com/s/mqs4e8z1lfut9l2/Screenshot%20%2858%29.png?dl=0
Which makes no sense, as there are no delays set between actions.

As a test I have just Cloned the main campaign to see if there was an issue with the Campaign itself.
The first stage is to Verify the Email address, which processed 100% ok.
IMAGE: https://www.dropbox.com/s/x2xdlyc7amp36um/Screenshot%20%2859%29.png?dl=0

The next step is to send the 1st Email. In total there are 4754 email to send to. However only 14 got processed.
IMAGE: https://www.dropbox.com/s/dl0i54jxrri9e8n/Screenshot%20%2860%29.png?dl=0

I can confirm that those 14 got Emailed out via Mailgun Successfully.
I have checked the logs and am getting no errors.

Looking at the SQL lead_event_log,  it is showing only the 14 sent with a Channel and Channel_id, and the ip_id' s are different for those 14 as well.

IMAGE: https://www.dropbox.com/s/fiy3jcuywe09o69/mau8g_campaign_lead_event_log.pdf?dl=0
Not sure if this is  any help but its an anomaly.

#############
Also why is every pending queue showing 4754. Surely the pending queue should only show a number when the task is actually called for. (i.e. you shouldnt be able to be in both the "Yes" and "No" queue, because only one can be the correct path).
#############
## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.14.0
| PHP version | 5.6.37
### Steps to reproduce:
1. 
2. 
 
### Log errors: 

LOGS ARE EMPTY

**Comment by Carlos-mb** [comment]
![image](https://user-images.githubusercontent.com/39302320/46008926-607e2880-c0be-11e8-9148-87fb784c04b1.png)

**Comment by Carlos-mb** [comment]
I have delete all conditions and left the campaing as a simple dip campign... it doesn't work

**Comment by Carlos-mb** [comment]
@iwantaspeaker  Did you change the trigger date or emails? 

I'm thiking about this as the origin of the issue in my case.

**Comment by iwantaspeaker** [comment]
Fixed in 2.14.1

**Comment by Carlos-mb** [comment]
I am running 2.14.1 and its not fixed.
The image I send is of 2.14.1

**Comment by eschiendorfer** [comment]
@Carlos-mb Could you find a solution for it? I seem to have the same problem...

**Comment by Carlos-mb** [comment]
No

I had 40 emails in one campaign. 

I have created 40 segments and 40 campaigns with 1 email each. Each segment has the contact to send each email to.

When I execute one of these campaign, their contacts go to the next segment and campaign. 

Then... I have created 5 campaign with 8 emails each. 

Now, every contact is in a grouped campaign (one of those with 8 emails). When a campaign fails, I use the single-email-campaign to move the contacts to the next "virgin" campaign.

Easy (Set sarcasm mode off)
